readme.txt

```diff
@@ -207,6 +207,7 @@ Our Premium Plugins:
 * Tweak - Move plugin CSS to PostCSS
 * Fix - Target specific input fields when check for remaining tickets to help with theme compatibility issues such as Avada (Thanks to Michael C!)
 * Fix -  Corrects capabilities test in relation to editing tickets (Props to @bokorir!)
+* Fix -  Loads thickbox for email modal in attendees list
```

Comment:
Weird spacing (line above too, though I realize you didn't add that one). Otherwise good to merge!